### DISCUSSION TITLE
Add ability to create and manage gists

### DISCUSSION THREAD

**Discussion by dmusicant-dk** [community/community#3331] [Mobile] [answered]
The mobile app has no way to view, create or manage gists. This would be very useful, can you please add this feature?

**Comment by eliperkins** [comment, marked as answer]
[We hear ya, @dmusicant-dk](https://github.com/github/feedback/discussions/3229). Viewing Gists within the app is on our radar. For now, best bet is to use https://gist.github.com within your mobile browser.

We'll keep you posted on this!

**Reply by dmusicant-dk** [reply]
Thank you @eliperkins!

**Reply by cpintoval** [reply]
+1 for this feature request! It would be extremely helpful to quickly edit a Gist directly from the mobile app. 

E.g. quickly fix a typo, add a comment/description, etc.

**Reply by alircxa** [reply]
Thanks a million! @eliperkins

**Reply by Adam-Kay** [reply]
Has there been any movement on this since it was last spoken about?

**Reply by abdelrahmanTD** [reply]
Are there any updates or still on radar for over **4 years**?

**Reply by diraneyya** [reply]
Many people, including myself, want this #18795

**Reply by Crapshit** [reply]
On your radar since 4 years? Still true?

**Reply by ThioJoe** [reply]
How are Gists ignored so badly 😔

![image](https://github.com/user-attachments/assets/f60a265e-3173-4337-83d4-164b8abc1955)

**Reply by Mustlane** [reply]
Still waiting.

**Reply by krampus-nuggets** [reply]
Usual corporate bs.

**Reply by pochopsp** [reply]
Four years later and still nothing.... Okay.

**Reply by brandon-fryslie** [reply]
What is the point of the app without gist support exactly?  I can open and submit an issue, which often have multistep requirements like posting error messages, validating configurations, and answering detailed questions, but I can’t take a block of code I copied and share it with someone?  Do you guys even internet over there?

Let’s do a thought experiment. Imagine GitHub had a “gist feed” and you could manage visibility, and each gist was kinda like a “post”.  You could see others gists and share code with people you know, and write a comment about it.  Tag people and whatnot, just like today.  What is fundamentally different than this and any social network?  

Now picture if the mobile Facebook or twitter app was like “yea you can change your time zone setting and edit group membership, but you can’t write a post.  No plans for it.  It’s on our radar tho”.

**Reply by copiliot-cmd** [reply]
2026 still nothing

**Comment by kanition** [comment]
Still nothing in 2025

**Comment by Dhyfer1** [comment]
5 years have passed since this issue was created and still there is no support for gists in the official app!? Incredible. How much longer must we continue waiting? 🙎
